build_tools_check.js

```diff
@@ -0,0 +1,27 @@
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+const verifyBuildToolInstallation = async (tool, checkCommand, installCommand) => {
+	try {
+		const { stdout } = await exec(checkCommand);
+		console.log(`${tool} is Installed: ${stdout}`);
+	} catch (error) {
+		console.warn(`WARNING: This development tool is not installed: "${tool}". Please install it using your package manager. For example: ${installCommand}`);
+	}
+};
+
+switch (process.platform) {
+case 'win32':
+	verifyBuildToolInstallation('Windows Build Tools', 'npm ls -g windows-build-tools', 'npm install --global windows-build-tools');
+	break;
+case 'darwin':
+	verifyBuildToolInstallation('CocoaPods', 'pod --version', 'sudo gem install cocoapods');
+	verifyBuildToolInstallation('rsync', 'rsync -version','apt-get install rsync');
+	break;
+case 'linux':
+	verifyBuildToolInstallation('rsync', 'rsync -version', 'apt-get install rsync');
+	break;
```

Comment:
There's no `apt-get` on macOS. rsync comes with macOS, but if you add a command add: `sudo port install rsync`

`sudo` is missing for the `apt-get` linux command.

Reply:
Please note that the flag is `--version` for rsync, not `-version`.

Reply:
Thanks for the change.

Reply:
Thanks for reviewing 💪🏾